src/hubmap/get-dataset-list.js

```diff
@@ -0,0 +1,52 @@
+import { getConfig } from '../util/common.js';
+
+const HUBMAP_TOKEN =
```

Comment:
**Never** commit tokens. You should place it in `env.sh` and then read it from the environment using the object returned by `getConfig()`